src/main/resources/assets/mythicmetals/lang/de_de.json

```diff
@@ -0,0 +1,1146 @@
+  "advancement.mythicmetals.maxed_royal_midas_gold_sword.title": "Vergoldetes Royal Overdrive",
+  "advancement.mythicmetals.maxed_gilded_midas_gold_sword.description": "Erreiche Maximallevel auf einem Vergoldetes Midas Gold Schwert",
+  "advancement.mythicmetals.maxed_gilded_midas_gold_sword.title": "Fünfzig Millionen Blöcke Later",
+  "advancement.mythicmetals.metallurgium.description": "Schmiede Metallurgium aus Hallowed, Palladium und Unerhaltbarium",
+  "advancement.mythicmetals.metallurgium.title": "Unbewegbares Objekt",
+  "advancement.mythicmetals.midas_gold.description": "Finde Midas Gold im Nether, welches jedes Erz in Gold verwandelt",
+  "advancement.mythicmetals.midas_gold.title": "GOLD!!!",
+  "advancement.mythicmetals.midas_gold_folding.description": "Crafte ein Midas Gold Falt Template aus Midas Gold and Netherrack, welches für Upgrades des Midas Gold Schwert benötigt wird.",
+  "advancement.mythicmetals.midas_gold_folding.title": "Professioneller Serviertenfalter",
+  "advancement.mythicmetals.royal_midas_smithing_template.description": "Tausche Gold mit Midas bis du ein Royal Midas Gold Schwert Template erhältst.",
+  "advancement.mythicmetals.royal_midas_smithing_template.title": "Ein wahrhaft edler Schwertsschmied",
+  "advancement.mythicmetals.midas_gold_sword.description": "Crafte ein Midas-Gold-Schwert, für dessen Aufwertung eine Menge Midas-Gold benötigt wird.",
+  "advancement.mythicmetals.midas_gold_sword.title": "Königliches Besteck",
+  "advancement.mythicmetals.mythril.description": "Finde Mythril, ein geheiligtes Erz, das in den oberen Teilen von Deepslatehöhlen vorkommt",
+  "advancement.mythicmetals.mythril.title": "Zwergenschatz",
+  "advancement.mythicmetals.mythril_drill_template.description": "Finde einen Mythril Bohrer Schmieden Template in verlassenen Minenschächten",
+  "advancement.mythicmetals.mythril_drill_template.title": "Versteckt im Gebirgsspeicher",
+  "advancement.mythicmetals.mythril_drill.description": "Schmiede einen Mythril Bohrer aus einer Durasteel Engine und einer Mythril Spitzhacke",
+  "advancement.mythicmetals.mythril_drill.title": "Graben und Graben macht uns frei",
+  "advancement.mythicmetals.orichalcum.description": "Finde Orichalcum, ein Hallowederz verstreut in den Deepslate Höhlen",
+  "advancement.mythicmetals.orichalcum.title": "Orkische Stärke",
+  "advancement.mythicmetals.orichalcum_hammer.description": "Erstelle einen Orichalcum Hammer, mit dem du schneller lange Tunnel graben kannst.",
+  "advancement.mythicmetals.orichalcum_hammer.title": "Du weißt, wie spät es ist",
+  "advancement.mythicmetals.osmium.description": "Finde Osmium hoch oben in den Bergen",
+  "advancement.mythicmetals.osmium.title": "Wann kann ich ein Windrad bauen?",
+  "advancement.mythicmetals.palladium.description": "Finde Palladium, normalerweise unter dem Lava-Ozean im Nether",
+  "advancement.mythicmetals.palladium.title": "Heißeste Baked Beans",
```

Comment:
Baked Beans is not translated here.

Reply:
Backed Beans could be translated to "Gebackene Bohnen", but that meal is also known as Baked Beans in Germany.